src/main/java/org/traccar/WebDataHandler.java

```diff
@@ -39,13 +47,17 @@
 import java.util.Formatter;
 import java.util.Locale;
 import java.util.TimeZone;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicInteger;
 
 @ChannelHandler.Sharable
 public class WebDataHandler extends BaseDataHandler {
 
     private static final String KEY_POSITION = "position";
     private static final String KEY_DEVICE = "device";
 
+    private static final Logger LOGGER = LoggerFactory.getLogger(WebDataHandler.class);
```

Comment:
This should be the first line in the class.

Reply:
Ok, I agree.